lib/_stream_transform.js

```diff
@@ -82,6 +82,10 @@ function TransformState(stream) {
   this.writeencoding = null;
 }
 
+function emitMultipleCallsError() {
+  this.emit('error', new Error('write callback called multiple times'));
+}
+
```

Comment:
I'm good with this, but it may make sense at some point to make this a utility in `internal/util` given that there may be several places where we'd want to avoid calling a callback more than once